COMETwebapp/Components/Viewer/PropertiesPanel/PropertiesComponent.razor

```diff
@@ -39,11 +39,11 @@ Copyright (c) 2023 RHEA System S.A.
                 @foreach (var parameter in parameters)
                 {
                     var classNames = parameter == this.ViewModel.SelectedParameter ? "parameter-item parameter-item-selected" : "parameter-item";
-                    <p class="@classNames" @onclick="()=> this.ViewModel.OnParameterClick(parameter)">@parameter.ParameterType.Name</p>
+                    <p class="@classNames" @onclick="()=> this.ViewModel.SelectedParameter = parameter">@parameter.ParameterType.Name</p>
```

Comment:
i would be a bit protective here and do something like?

@parameter.ParameterType?.Name (you never know what the server returns right, ParameterType could be null